package.json

```diff
@@ -646,6 +632,7 @@
     "webpack": "^4.44.1"
   },
   "dependencies": {
```

Comment:
Can we remove dependencies like `serialport` now? There may also be webpack logic that can be simplified.